src/pydap/model.py

```diff
@@ -258,6 +265,10 @@ def __init__(
     def __repr__(self):
         return "<%s with data %s>" % (type(self).__name__, repr(self.data))
 
+    @property
```

Comment:
This is useful when the variable names is a path-like (i.e. groups, sequences, etc)